scripts/eqtl_hail_batch/generate_eqtl_spearman.py

```diff
@@ -175,6 +177,13 @@ def create_struct(gene):
     data_summary = data_summary.rename({'index': 'gene_symbol'}, axis='columns')
     # add in cell type info
     data_summary['cell_type_id'] = celltype
+    # add in ENSEMBL IDs
+    gtf = pd.read_csv(GENCODE_GTF, sep='\t', header=None, index_col=False, skiprows=5)
```

Comment:
Would it be easier to use https://hail.is/docs/0.2/experimental/index.html#hail.experimental.import_gtf here instead?

Reply:
Doesn't that get imported as a hail table? I only need a pandas df here

Reply:
Yes, but you could easily convert the Hail Table to a Pandas DataFrame. You then wouldn't need any of that parsing code below.

Reply:
Would you happen to know how to import the GTF? Using the same GTF file I specified at the top of the code and loading in using
```
hl.experimental.import_gtf(GENCODE_GTF, reference_genome='GRCh38', skip_invalid_contigs=True, force_bgz=True)
```
I get 
```
Error summary: ZipException: File does not conform to block gzip format.
```

Reply:
I think this is still pending on the discussion here: https://centrepopgen.slack.com/archives/C018KFBCR1C/p1661410430973899?thread_ts=1658208231.938539&cid=C018KFBCR1C